frontend/packages/kubevirt-plugin/src/utils/url.ts

```diff
@@ -95,6 +96,13 @@ export const getVMWizardCreateLink = ({
 }) => {
   const params = new URLSearchParams();
   const initialData: VMWizardInitialData = {};
+  const isSapHanaTemplate = template?.metadata?.labels[TEMPLATE_SAP_HANA] === 'true';
+
+  // TODO - Instantiate template
```

Comment:
the todo is to render the new initiate template page for all templates instead of the current wizards, but this will happen only in 4.11, no need to state a TODO here, we will get to remove the current wizards (simple and customize) it in the next step.